terraform/digitalocean.sample.tf

```diff
@@ -13,6 +13,6 @@ module "do-hosts" {
   region_name = "nyc3" # this must be a region with metadata support
 
   control_count = 3
-  worker_count = 3
+  worker_count = 4
```

Comment:
It's likely not strictly necessary on all clouds - it depends on a combination of the number of nodes and the image types (how much ram and cpu). I think 4 nodes covers it on all clouds I have tested so far and was just trying to keep things consistent.